posthog/test/test_client.py

```diff
@@ -106,6 +106,40 @@ def test_basic_capture_with_feature_flags(self, patch_decide):
 
         self.assertEqual(patch_decide.call_count, 1)
 
+    @mock.patch("posthog.client.decide")
+    def test_get_active_feature_flags(self, patch_decide):
+        patch_decide.return_value = {
+            "featureFlags": {"beta-feature": "random-variant", "alpha-feature": True, "off-feature": False}
+        }
+
+        client = Client(FAKE_TEST_API_KEY, on_error=self.set_fail, personal_api_key=FAKE_TEST_API_KEY)
+        variants = client._get_active_feature_variants("some_id", None, None, None)
+        print(variants)
```

Comment:
I know this is already merged but is this print meant to be here? There's no test assertion statement in this test

Reply:
Woops!

Reply:
fixed here: https://github.com/PostHog/posthog-python/pull/90